test/run-blackbox-tests-interactive.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+echo "Refreshing test images"
+docker compose \
+  -f "$DIR/blackbox/docker-compose.yaml" \
+  build
+
+echo "Starting interactive blackbox test runner shell"
+echo "Hint: Run docker compose logs -f in another terminal to see what's happening"
+echo "Hint: Changes to tests are mounted live, but changes to application code *will not* apply"
+echo "      until you re-build the containers (e.g. by exiting and re-running this script)"
```

Comment:
Would using https://docs.docker.com/compose/file-watch/ help with not having to manually quit, teardown, rebuild and run?

Reply:
Not as-is, because you also need to commit changes to the `src` to have `git archive` include them. It would be possible to have the docker build do a `git stash` and then archive the stash rather than HEAD but would only want to do that in dev so it means more flags and complexity to the build process... Will clarify why not in the docs.